rusk-prover/Cargo.toml

```diff
@@ -4,14 +4,22 @@ version = "0.3.0"
 edition = "2021"
 autobins = false
 
-
 [dependencies]
 dusk-bytes = { version = "0.1" }
-dusk-wallet-core = { version = "0.25.0-phoenix.0.26", default-features = false }
+phoenix-core = { version = "0.26", default-features = false, features = ["alloc", "rkyv-impl"] }
+dusk-jubjub = { version = "0.14", default-features = false, features = ["rkyv-impl"] }
+dusk-plonk = { version = "0.19", default-features = false }
+jubjub-schnorr = { version = "0.2", default-features = false, features = ["double","rkyv-impl"] }
```

Comment:
can you depend on `execution-core` and use the re-exported types from there instead?